package.json

```diff
@@ -28,14 +28,14 @@
     "signal-exit": "^3.0.3",
     "spawnd": "^4.4.0",
     "tree-kill": "^1.2.2",
-    "wait-on": "^5.2.0",
-    "@types/wait-on": "^5.2.0"
```

Comment:
I think we should keep it in there because if the user installs it, only the dependencies get installed. But since we refer to it in the types.d.ts it will lead to errors for the wait-on package since the types are not installed. Or am I wrong?

Reply:
You're right, I missed that. I originally was trying to pull the values from this fix from @types/wait-on but it doesn't look like they exist there. I've backed out the npm changes